packages/turf-angle/test.js

```diff
@@ -66,3 +67,15 @@ test('turf-angle -- issues', t => {
     t.false(isNaN(a), 'result is not NaN');
     t.end();
 });
+
+test('turf-angle -- throws', t => {
+    const pt1 = point([-10, -30]);
+    const pt2 = point([-11, -33]);
+    const pt3 = point([-12, -36]);
+    t.throws(() => angle(null, pt2, pt3), /obj is required/, 'missing startPoint');
```

Comment:
Maybe we should throw an Error that says `missing startPoint` instead of `obj is required`

Reply:
Yes, I was concerned about the performance, but it should not be a big deal.
Have you checked the benchmark by any chance?

Reply:
I wouldn't worry about performance for those types of checks, it doesn't slow anything down.

The biggest performance hits is reiterating over a FeatureCollection twice when 1 pass can achieve the same result.

I'd say all the input type checking is always valid to add, the "heavy" performance hit will be the main logic and not the input checking.